### PR TITLE
Feat/better contact section

### PR DESCRIPTION
**Pull request by jnizet**

### CODE REVIEW

File: frontend/src/app/person/person.component.spec.ts [hunk 1]
```diff
@@ -42,7 +42,7 @@ describe('PersonComponent', () => {
     abroadFamilySituation: null
   };
 
-  const activatedRoute = {
+  const activatedRoute: any = {
```
Comment (cexbrayat) [inline frontend/src/app/person/person.component.spec.ts]: nitpicking `const activatedRoute = { ... } as ActivatedRoute;` would be better IMO

Comment (jnizet) [inline frontend/src/app/person/person.component.spec.ts]: I would love to, but it doesn't compile. I don't know exactly why. I always thought of 'as Foo' to mean "yeah, I know it's not that type, but don't bother", but that doesn't actually work that way:

```
ERROR in /Users/jb/projects/globe42/frontend/src/app/person/person.component.spec.ts (45,26): Type '{ parent: { snapshot: { data: { person: PersonModel; }; }; }; }' cannot be converted to type 'ActivatedRoute'.
  Property 'url' is missing in type '{ parent: { snapshot: { data: { person: PersonModel; }; }; }; }'.
```

Comment (cexbrayat) [inline frontend/src/app/person/person.component.spec.ts]: Out of curiosity, I tried to make it compiles. 
This issue is with `data`, you have to write:

```typescript
  const activatedRoute = {
    parent: {
      snapshot: { data: { person } as Data }
    }
  } as ActivatedRoute;
```
A bit ugly, but works.

**Comment by acrepet** [comment]
LGTM !

**Pull request merged**